pkg/cidata/cidata.TEMPLATE.d/boot/30-install-packages.sh

```diff
@@ -57,6 +57,23 @@ elif command -v pacman >/dev/null 2>&1; then
 		fi
 	fi
 	# other dependencies are preinstalled on Arch Linux (https://linuximages.de/openstack/arch/)
+elif command -v zypper >/dev/null 2>&1; then
+	if [ "${LIMA_CIDATA_MOUNTS}" -gt 0 ]; then
+		if ! command -v sshfs >/dev/null 2>&1; then
+			zypper install -y sshfs
+		fi
+	fi
+	if [ "${LIMA_CIDATA_CONTAINERD_SYSTEM}" = 1 ] || [ "${LIMA_CIDATA_CONTAINERD_USER}" = 1 ]; then
+		if [ ! -e /usr/sbin/iptables ]; then
+			zypper install -y iptables
+			modprobe ip_tables
```

Comment:
modprobe must always happen here (even when /usr/sbin/iptables is already present.)
https://github.com/AkihiroSuda/lima/blob/eb3ef88dd3a9834ea15cec99e1be2073c4c4b16c/pkg/cidata/cidata.TEMPLATE.d/boot/40-install-containerd.sh#L28

Reply:
Ok, I've added it just after the `modprobe tap` line. I assume for `modprobe ip_tables` we don't need the `|| true` part because a failure would be fatal?

Reply:
We can have `|| true` because ip_tables can be built-in into the kernel